docs/book/src/topics/crd-changes/v1alpha6-to-v1alpha7.md

```diff
@@ -218,17 +218,40 @@ profile:
 
 We now have the ability for a machine to have additional block devices to be attached.
 
-Here is an example on how to use `additionalBlockDevices` for adding additional Cinder volumes attached
+Here is an example on how to use `additionalBlockDevices` for adding an additional Cinder volume attached
 to the server instance:
 
 ```yaml
 additionalBlockDevices:
 - name: database
-  size: 50
+  type: volume
+  diskSize: 50
   volumeType: my-volume-type
   availabilityZone: az0
 ```
 
+Here is an example on how to attach two ephemeral disks to the instance, where the second disk will be formated to `ext4`:
+
+```yaml
+- name: disk1
+  type: local
+  diskSize: 1
+- name: disk2
+  type: local
+  diskSize: 1
+  guestFormat: ext4
```

Comment:
We probably want to document that `guestFormat` should be used at user's own risks. Nova formatted ephemeral disks have always been problematic.